server/channels/app/imports/import_validators_test.go

```diff
@@ -710,14 +710,6 @@ func TestImportValidateBotImportData(t *testing.T) {
 	data.DisplayName = model.NewPointer("Display Name")
 
 	// Invalid Owner Name.
-	data.Owner = nil
```

Comment:
Can we keep this as a negative test to verify that no error is returned if Owner is nil?